kong/llm/drivers/llama2.lua

```diff
@@ -253,18 +255,16 @@ function _M.post_request(conf)
 end
 
 function _M.pre_request(conf, body)
```

Comment:
_Continues: https://github.com/Kong/kong/pull/12807/files#r1565677465_

@hanshuebner 

This was a mistake, and it's fixed now. The check is removed from ALL drivers.